index.html

```diff
@@ -139,15 +139,7 @@ <h1 class="about-title">About me</h1>
            </p>
            <a href="#" class="one btn btn2">Get My Resume</a>
          <div class="a-design">
-            <img src="img/v15.png" class="block1">
-            <img src="img/v16.png" class="block2">
-            <img src="img/v16.png" class="block3">
-            <img src="img/v17.png" class="block4">
-            <img src="img/v19.png" class="block5">
-            <img src="img/v20.png" class="block6">
-            <img src="img/v21.png" class="block7">
-            <img src="img/v22.png" class="block8">
-            <img src="img/v23.png" class="block9">
+            <img src="img/Group (1).png" class="block1">
          </div>
```

Comment:
- [Optional] Please use an indentation of two spaces in all your documents. This is a convention and also best practice.